### DISCUSSION TITLE
Adding delete option for unwanted components.

### DISCUSSION THREAD

**Discussion by Harshini2411** [Atri-Labs/atrilabs-engine#594] [Discuss: New Feature Ideas]
Often we select the buttons, text-boxes or some other components on the editor window. Suppose if we want to remove one of them from the editor instantly, there is none like that. So, I suggest you to please add that delete or cross-mark to the components,

**Comment by cruxcode** [comment]
@Harshini2411 You can delete a component by first selecting the component and then pressing Delete key on your keyboard.

**Reply by Harshini2411** [reply]
Okk
